sdk/agrifood/agrifood-farming-rest/review/agrifood-farming.api.md

```diff
@@ -7,107 +7,169 @@
 import { Client } from '@azure-rest/core-client';
 import { ClientOptions } from '@azure-rest/core-client';
 import { HttpResponse } from '@azure-rest/core-client';
-import { PagedAsyncIterableIterator } from '@azure-rest/core-client-paging';
+import { LroEngineOptions } from '@azure/core-lro';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PathUncheckedResponse } from '@azure-rest/core-client';
-import { PollerLike } from '@azure-rest/core-client-lro';
-import { PollerOptions } from '@azure-rest/core-client-lro';
-import { PollOperationState } from '@azure-rest/core-client-lro';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
 import { RequestParameters } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public (undocumented)
 export interface ApplicationData {
-    applicationProductDetails?: ApplicationProductDetail[];
+    applicationProductDetails?: Array<ApplicationProductDetail>;
     area?: Measure;
     associatedBoundaryId?: string;
     attachmentsLink?: string;
     avgMaterial?: Measure;
-    createdDateTime?: Date;
+    createdDateTime?: Date | string;
     description?: string;
     eTag?: string;
     farmerId?: string;
     id?: string;
-    modifiedDateTime?: Date;
+    modifiedDateTime?: Date | string;
     name?: string;
     operationBoundaryId?: string;
-    operationEndDateTime?: Date;
-    operationModifiedDateTime?: Date;
-    operationStartDateTime?: Date;
-    properties?: ApplicationDataPropertiesDictionary;
+    operationEndDateTime?: Date | string;
+    operationModifiedDateTime?: Date | string;
+    operationStartDateTime?: Date | string;
+    properties?: Record<string, Record<string, unknown>>;
```

Comment:
So it's a property bag of property bags? The previous type was just Record<string, unknown> so why the additional nesting?